Index: smartanki.egg-info/SOURCES.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>setup.py\r\nsmartanki/__init__.py\r\nsmartanki/anki_export.py\r\nsmartanki/cefr_filter.py\r\nsmartanki/extractor.py\r\nsmartanki/main.py\r\nsmartanki/main_cli.py\r\nsmartanki/pdf_reader.py\r\nsmartanki/translator.py\r\nsmartanki/utils.py\r\nsmartanki/vocab_db.py\r\nsmartanki.egg-info/PKG-INFO\r\nsmartanki.egg-info/SOURCES.txt\r\nsmartanki.egg-info/dependency_links.txt\r\nsmartanki.egg-info/entry_points.txt\r\nsmartanki.egg-info/requires.txt\r\nsmartanki.egg-info/top_level.txt
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/smartanki.egg-info/SOURCES.txt b/smartanki.egg-info/SOURCES.txt
--- a/smartanki.egg-info/SOURCES.txt	(revision 4aed77aa36dca6ff3420688cc1a65b5e5e43f21a)
+++ b/smartanki.egg-info/SOURCES.txt	(date 1748354255985)
@@ -1,17 +1,25 @@
 setup.py
 smartanki/__init__.py
 smartanki/anki_export.py
+smartanki/anki_import.py
+smartanki/anki_package_export.py
 smartanki/cefr_filter.py
+smartanki/dictionary_api.py
 smartanki/extractor.py
 smartanki/main.py
 smartanki/main_cli.py
+smartanki/pdf_export.py
 smartanki/pdf_reader.py
+smartanki/test_runner.py
 smartanki/translator.py
 smartanki/utils.py
 smartanki/vocab_db.py
+smartanki/wordnet_backup.py
 smartanki.egg-info/PKG-INFO
 smartanki.egg-info/SOURCES.txt
 smartanki.egg-info/dependency_links.txt
 smartanki.egg-info/entry_points.txt
 smartanki.egg-info/requires.txt
-smartanki.egg-info/top_level.txt
\ No newline at end of file
+smartanki.egg-info/top_level.txt
+smartanki/agrosmodel/__init__.py
+smartanki/fonts/__init__.py
\ No newline at end of file
Index: smartanki.egg-info/PKG-INFO
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Metadata-Version: 2.4\r\nName: smartanki\r\nVersion: 0.1.0\r\nSummary: \uD83D\uDCD8 SmartAnki – Extract CEFR-based vocabulary and generate Anki flashcards from text or PDF.\r\nAuthor: Dmitry Yarochkin\r\nRequires-Python: >=3.10\r\nRequires-Dist: spacy\r\nRequires-Dist: nltk\r\nRequires-Dist: requests\r\nRequires-Dist: pdfplumber\r\nRequires-Dist: beautifulsoup4\r\nDynamic: author\r\nDynamic: requires-dist\r\nDynamic: requires-python\r\nDynamic: summary\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/smartanki.egg-info/PKG-INFO b/smartanki.egg-info/PKG-INFO
--- a/smartanki.egg-info/PKG-INFO	(revision 4aed77aa36dca6ff3420688cc1a65b5e5e43f21a)
+++ b/smartanki.egg-info/PKG-INFO	(date 1748354256010)
@@ -1,4 +1,4 @@
-Metadata-Version: 2.4
+Metadata-Version: 2.1
 Name: smartanki
 Version: 0.1.0
 Summary: 📘 SmartAnki – Extract CEFR-based vocabulary and generate Anki flashcards from text or PDF.
@@ -9,7 +9,3 @@
 Requires-Dist: requests
 Requires-Dist: pdfplumber
 Requires-Dist: beautifulsoup4
-Dynamic: author
-Dynamic: requires-dist
-Dynamic: requires-python
-Dynamic: summary
